km62/Shumel_Sofiia/laboratory1.sql

```diff
@@ -1,2 +1,149 @@
 -- LABORATORY WORK 1
 -- BY Shumel_Sofiia
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+створювати таблиці та видаляти з них дані
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER STUDENT IDENTIFIED BY pass
+DEFAULT TABLESPACE "USERS"
+TEMPORARY TABLESPACE "TEMP"
+QUOTA 50M FROM "USERS"
+
+GRANT CREATE ANY TABLE FOR STUDENT_NAME
+GRANT DROP FOR STUDENT_NAME
+
+
+
+
+
+
+
+
+
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Студент здає роботу викладачу.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+CREATE TABLE student
```

Comment:
Table name should be plural. Please rename student to student**s**.

Reply:
Regular expressions:
- name_student ^[A-Z]{1}[a-z]{1,20} [A-Z]{1}[a-z]{1,20}$
- id_student ^[0-9]+$
- course_student ^[1-6]$